web/public/images/electricitymap-icon.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
didn't we already have an icon?

Reply:
no, this is just the emap symbol as svg, we only had emap smybol + text as svg

Reply:
ok!